app/models/partner.rb

```diff
@@ -13,7 +13,26 @@ def ensure_role
     self.role = 'partner'
   end
 
+  def balance
+    wallet.contribution_balance.amount_bat + invoice_amount
+  end
+
+  def balance_in_currency
+    wallet.contribution_balance.add_bat(invoice_amount)
+
```

Comment:
nit: maybe we can rm the whitespace on `line 22`

Reply:
https://github.com/brave-intl/publishers/pull/1667/commits/092d85279ad37ad6a8ce82db79c7972594e84ead